test/e2e/specs/record/helpers.js

```diff
@@ -167,7 +167,12 @@ exports.testPresentation = function (tableParams) {
                     // verify all rows are present
                     chaisePage.recordPage.getRelatedTableRows(displayName).count().then(function(rowCount) {
                         expect(rowCount).toBe(relatedTables[i].data.length);
-                        expect(headings[i]).toBe("- " + displayName + " (" + rowCount + ")");
+                        // The annotation_image table has more rows than the page_size, so its heading will have a + after the row count
+                        if (displayName == tableParams.related_table_name_with_page_size_annotation) {
+                            expect(headings[i]).toBe("- " + displayName + " (" + rowCount + "+)");
+                        } else {
+                            expect(headings[i]).toBe("- " + displayName + " (" + rowCount + ")");
+                        }
```

Comment:
I'm guessing this function in helpers is being used for multiple specs? This is a case of a test that looks to be non-deterministic and should be written as 2 separate ones :/